ray-operator/config/samples/ray-cluster.tpu-v6e-256-multihost.yaml

```diff
@@ -67,6 +67,8 @@ spec:
             ports:
             - containerPort: 8081
               name: mxla
+            securityContext:
+              privileged: true
```

Comment:
Is there any possible solution? Setting `privileged` to true does not seem to be acceptable to most users.

Reply:
This was fixed with a newer version of the TPU device plugin, which is shipped in GKE version 1.31.1-gke.1678000. I'll remove this section from the manifest.